src/Printer.ts

```diff
@@ -0,0 +1,38 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+const filePath: string = "/home/user/Desktop/Node/DirectoryAndFilePrinter/node_modules"
```

Comment:
I would suggest to avoid using hardcoded path, also this path will work only for your machine.
Please use `path` module to replace hardcoded path.